pySDC/implementations/problem_classes/DiscontinuousTestODE.py

```diff
@@ -0,0 +1,216 @@
+import numpy as np
+
+from pySDC.core.Errors import ParameterError, ProblemError
+from pySDC.core.Problem import ptype
+from pySDC.implementations.datatype_classes.mesh import mesh
+
+
+class DiscontinuousTestODE(ptype):
+    r"""
+    This class implements a very simple test case of a ordinary differential equation consisting of one discrete event. The dynamics of
+    the solution changes when the state function :math:`h(u) := u - 5` changes the sign. The problem is defined by:
+
+    if :math:`u - 5 < 0:`
```

Comment:
Why this choice of $5$ for the switch ? Cannot it be a parameter of the problem ?

Reply:
It is chosen arbitrarily since it is a self-constructed problem. The exact event time and also everything based on this value.